### ISSUE TITLE
CompiledName not working for certain cases 

### ISSUE THREAD

**Issue by thautwarm** [fable-compiler/Fable.Python#40]
```F#
[<CompiledName("toNum")>]
let toNum (x: string) = decimal x

[<CompiledName("Num")>]
let num (x: string) = decimal x
```

The first function compiles into `to_num`, which is unexpected. The second compiles into `Num`.

**Comment by dbrattli** [comment]
Thanks for submitting. This will be fixed in the next release.

**Comment by thautwarm** [comment]
fix comfirmed in 4.0.0-alpha-026.
